x-pack/plugins/observability/public/embeddable/slo/overview/slo_embeddable.tsx

```diff
@@ -50,8 +50,15 @@ export class SLOEmbeddable extends AbstractEmbeddable<SloEmbeddableInput, Embedd
     this.updateInput({ title });
   }
 
+  public onRenderComplete() {
+    this.renderComplete.dispatchComplete();
+  }
+
   public render(node: HTMLElement) {
+    super.render(node);
     this.node = node;
+    // required for the export feature to work
+    this.node.setAttribute('data-shared-item', '');
```

Comment:
This is needed according to [this](https://github.com/elastic/kibana/blob/main/x-pack/plugins/aiops/public/embeddable/embeddable_change_point_chart.tsx#L113). Otherwise reporting wouldn't work